src/lib/pci_virtio_sock.c

```diff
@@ -130,6 +130,11 @@ static int pci_vtsock_debug = 0;
 
 /* XXX need to use rx and tx more consistently */
 
+struct vtsock_config_hdr {
+	uint32_t len;
+	uint32_t version;
+};
```

Comment:
The `type` member was added in a4ec860c5600d43b3103e7e7a9394bcc6a070fd6 but I subsequently rewrote the initial protocol commit to reflect the intent. I can fix this up if you think it would improve clarity.